resources/lang/en/people.php

```diff
@@ -276,5 +276,7 @@
     'debt_add_amount' => 'the sum of',
     'debt_add_reason' => 'for the following reason (optional)',
     'debt_add_add_cta' => 'Add debt',
+    'debt_edit_update_cta' => 'Update debt',
```

Comment:
No problem, you're not annoying at all, I actually appreciate you taking the time to explain these things so I'll know in the future!
I've added the missing translations in a new commit.